lib/modules/manager/terraform/extractors/resources/helm-release.ts

```diff
@@ -67,4 +73,18 @@ export class HelmReleaseExtractor extends DependencyExtractor {
 
     return dependencies;
   }
+
+  private processOci(
+    depName: string,
```

Comment:
Why do you not extract `depName` from `dep`?

Reply:
it would change existing tests